src/plugins/analysis/hash/test/test_plugin_hash.py

```diff
@@ -23,20 +31,51 @@
 @pytest.mark.AnalysisPluginTestConfig(plugin_class=AnalysisPlugin)
 class TestAnalysisPluginHash:
     def test_all_hashes(self, analysis_plugin):
-        result = analysis_plugin.process_object(MockFileObject()).processed_analysis[analysis_plugin.NAME]
+        with TEST_FILE.open('rb') as fp:
+            result = analysis_plugin.analyze(fp, {}, ANALYSIS_RESULT)
 
-        assert 'md5' in result, 'md5 not in result'
-        assert 'sha1' in result, 'sha1 not in result'
-        assert 'foo' not in result, 'foo in result but not available'
-        assert result['md5'] == '6f8db599de986fab7a21625b7916589c', 'hash not correct'
-        assert 'ssdeep' in result, 'ssdeep not in result'
-        assert 'imphash' in result, 'imphash not in result'
+        assert isinstance(result.md5, str)
+        assert isinstance(result.sha1, str), 'sha1 not in result'
+        assert isinstance(result.ssdeep, str), 'ssdeep not in result'
+        assert isinstance(result.imphash, str), 'imphash not in result'
+        assert not hasattr(result, 'foo')
```

Comment:
This is redundant, isn't it?
Also, what was this supposed to test in the first place?! Someone adding a hash named `foo` accidentally?